scripts/benchmarks_ci.py

```diff
@@ -228,6 +232,24 @@ def __is_valid_datetime(dt: str) -> str:
         help='Partition Index of the run',
     )
 
+    parser.add_argument(
+        '--enable-open-telemetry-logger',
+        dest='enable_open_telemetry_logger',
+        required=False,
+        default=False,
+        action='store_true',
+        help='Enables OpenTelemetry logging',
+    )
+
+    parser.add_argument(
```

Comment:
It looks like we never actually use this argument any where. Am I missing something?